hasher-matcher-actioner/hmalib/fetching/tests/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Not part of the hmalib public interface, however, because of how
+# SignalExchangeAPI classes are handled, we need this to be reachable by python
```

Comment:
Confused about why this comment is here.
Python packaging expects an `__init__.py` if you want to import any files from this folder, I don't think this has anything to do with SignalExchangeAPI.

Reply:
Typically, we don't make tests findable. Unless it is a case of re-use. 

I would not have needed to add the tests in the `hmalib.fetching.tests` module to the hmalib package, but because of the way SignalExchangeAPI works, I had to bite the bullet and add it in. 

This is not new, however, we have done it earlier to make some test utils re-useable. Anyhoo. Can revise this comment.